### PR TITLE
Adding Resources page to the landing

### PR DESCRIPTION
**Pull request by znegrin**
This PR is related to the issue: https://github.com/src-d/backlog/issues/1404

It adds the Resources page to the landing:

image

Signed-off-by: Zuri Negrín <zurinegrin@gmail.com>

### CODE REVIEW

**Comment by dpordomingo** [comment]
available at staging
https://landing-staging.srcd.run/resources

got two conflicts during merge, and decided the following:
>~`$medium-light:  #9F9F9F;`~
>`$medium-light:  #989898;`

and
>~`$purple:        #d210c9;`~
>`$purple:        #8719cb;`

**Comment by dpordomingo** [comment]
In mobile, some buttons and headings are too close to some images.
image
image
image

**Comment by znegrin** [comment]
I pushed a fix for the mobile version. It should be fine now:

image

File: hugo/layouts/resources/single.html [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+</div>
+</header>
+
+<section id="documentation" class="pt-5 pb-0">
+    <div class="container">
+        <div class="row text-dark">
+            <div class="col-md-6">
+                <h2 class="title-line h3 line-yellow">source{d} Documentation</h2>
+                <p>source{d} exposes powerful Universal ASTs to analyze your code and a SQL engine to analyze your git history. Check out our documentation to get started, learn about advanced SQL queries and the underlying components that composed our product architecture.</p>
+                <a href="https://docs.sourced.tech/" class="btn btn-highlight col-md-6 mt-3 mb-5">Documentation</a>
```
Comment (dpordomingo) [inline hugo/layouts/resources/single.html]: @vcoisne should we link these old docs?

**Comment by znegrin** [comment]
@vcoisne added the new title and the newsletter section:

image

image

**Comment by vcoisne** [comment]
LGTM 

Only one change for now: 

- Remove all bullet points under biweekly newsletter section

**Comment by znegrin** [comment]
Hi @vcoisne 

I changed the resources page as requested:

image

**Comment by dpordomingo** [comment]
imo this is ready to go

**Comment by znegrin** [comment]
I updated the Forum image as mentioned on #424

**Comment by vcoisne** [comment]
LGTM

**Pull request merged**